### PR TITLE
chore: update gradlew & openjdk

### PR DESCRIPTION
**Pull request by mull**
```bash
# from project root
mkdir ./data
mkdir ./packages/booking-dispatcher/data
docker compose up # voila!
```

I'm not exactly sure why, but gradlew 6.9 accompanied with a major version bump of JDK works on the new Apple M1 chips.

The gradle version was [updated as per the instructions](https://docs.gradle.org/6.9/release-notes.html)

### CODE REVIEW

File: packages/route-optimization-jsprit/gradlew [hunk 1]
```diff
@@ -129,6 +130,7 @@ fi
 if [ "$cygwin" = "true" -o "$msys" = "true" ] ; then
     APP_HOME=`cygpath --path --mixed "$APP_HOME"`
     CLASSPATH=`cygpath --path --mixed "$CLASSPATH"`
+
```
Comment (raduachim) [inline packages/route-optimization-jsprit/gradlew:133]:
```suggestion
```
unless needed

File: packages/route-optimization-jsprit/gradlew [hunk 2]
```diff
@@ -82,6 +82,7 @@ esac
 
 CLASSPATH=$APP_HOME/gradle/wrapper/gradle-wrapper.jar
 
+
```
Comment (raduachim) [inline packages/route-optimization-jsprit/gradlew:85]:
```suggestion
```
unless needed

Comment (mull) [reply packages/route-optimization-jsprit/gradlew:85]: Most likely not, it's just autogenerated by gradle itself

Comment (raduachim) [reply packages/route-optimization-jsprit/gradlew:85]: ah interesting :+1:

Comment (guen-ltu) [reply packages/route-optimization-jsprit/gradlew:85]: Thanks for helping out here! I was also running the tests in a container with OpenJDK17 and all tests got green.

**Review by guen-ltu — APPROVED** [review]

File: packages/route-optimization-jsprit/gradlew (hunk 2 above)
Comment (mull) [reply packages/route-optimization-jsprit/gradlew:85]: Sure, but you didn't want to merge it? You closed the PR :)

Comment (mull) [reply packages/route-optimization-jsprit/gradlew:85]: I think we need Gradle 7.0 for OpenJDK17, IIRC 16 is only supported by 6.9 and/or 7.0

**Comment by guen-ltu** [comment]
reopened because I forgot to merge :)

**Pull request merged**